lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -2,42 +2,41 @@
 """
 Tests of verify_student views.
 """
+
+import ddt
```

Comment:
This is a third-party import; could you please move it to the second block of imports?